### PR TITLE
Request money on Sandbox if balance zero. (bunq/tinker_csharp#7)

### PR DESCRIPTION
**Pull request by JordyHeemskerk**

### CODE REVIEW

File: TinkerSrc/Lib/BunqLib.cs [hunk 1]
```diff
@@ -214,13 +223,27 @@ private SandboxUser GenerateNewSandboxUser()
...
 
             var responseString = requestTask.Result.Content.ReadAsStringAsync().Result;
             var responseJson = BunqJsonConvert.DeserializeObject<JObject>(responseString);
             return BunqJsonConvert.DeserializeObject<SandboxUser>(responseJson.First.First.First.First.First
                 .ToString());
         }
+
+        private void RequestSpendingMoneyIfNeeded()
+        {
+            if (ApiEnvironmentType.SANDBOX.Equals(EnvironmentType)
```
Comment (OGKevin) [inline TinkerSrc/Lib/BunqLib.cs]: Logic can be extracted in its own separate method.

File: TinkerSrc/Lib/BunqLib.cs [hunk 2]
```diff
@@ -36,13 +36,22 @@ public class BunqLib
...
+
+        private const double BalanceZero = 0.0;
 
         private ApiEnvironmentType EnvironmentType { get; set; }
 
         public BunqLib(ApiEnvironmentType environmentType)
         {
             EnvironmentType = environmentType;
             SetupContext();
+            RequestSpendingMoneyIfNeeded();
```
Comment (OGKevin) [inline TinkerSrc/Lib/BunqLib.cs:56]: TODO: bunq/sdk_csharp#89

after sdk update.

**Review by OGKevin — CHANGES_REQUESTED** [review]
Minor changes.

**Comment by JordyHeemskerk** [comment]
@OGKevin moving UserContext refresh to new issue. Implemented comments.

**Review by OGKevin — APPROVED** [review]
👌

**Comment by OGKevin** [comment]
@andrederoos

**Pull request merged**